src/components/CreateInstance/CreateInstance.stories.tsx

```diff
@@ -0,0 +1,385 @@
+        getModalAppendTo={() => document.getElementById(id)}
+        isModalOpen={isModalOpen}
+        hideModal={onCloseModal}
+        disableFocusTrap={true}
+
+      />
+      <div>
+        <Button onClick={() => onOpenModal()}>
+          Open modal
+          </Button>
+      </div>
+    </div>
+  );
+};
+
+const cloudProviders = [
+  {
+    display_name: "Amazon Web Services",
+    name: "aws",
+  },
+];
+
+const cloudRegions = [
+  {
+    id: "eu-west-1",
+    display_name: "EU, Ireland",
+  },
+  {
+    id: "us-east-1",
+    display_name: "US East, N. Virginia",
+  },
+];
+
+
+
+export const AllReady = Template.bind({});
+AllReady.args = {
+  isKasTrial: true,
+  cloudProviders: cloudProviders,
+  quota: { data: undefined, loading: false, isServiceDown: false },
+  //kasQuota:{allowed:60,consumed:20,remaining:40},
+  kasTrial: { allowed: 0, consumed: 0, remaining: 0 },
+  hasKafkaCreationFailed: false,
+  userHasTrialInstance: false,
+  loadingQuota: false,
+  isCreationInProgress: false,
+  kafkaRequest: {
+    cloud_provider: {
+      value: "aws",
+    },
+    multi_az: {
+      value: true,
+    },
+    region: {
+      value: "",
+    },
+    name: {
+      value: "",
+    },
+  },
+  formSubmitted: false,
+  cloudRegions: cloudRegions,
+};
+AllReady.storyName = "Ready to create a trial instance";
+
+export const QuotaLoading = Template.bind({});
+QuotaLoading.args = {
+  isKasTrial: true,
+  cloudProviders: cloudProviders,
+  quota: { data: undefined, loading: true, isServiceDown: false },
+  kasQuota: { allowed: 60, consumed: 20, remaining: 40 },
+  kasTrial: { allowed: 60, consumed: 20, remaining: 40 },
+  hasKafkaCreationFailed: false,
+  userHasTrialInstance: false,
+  loadingQuota: true,
+  isCreationInProgress: false,
+  kafkaRequest: {
+    cloud_provider: {
+      value: "aws",
+    },
+    multi_az: {
+      value: true,
+    },
+    region: {
+      value: "",
+    },
+    name: {
+      value: "",
+    },
+  },
+  formSubmitted: false,
+  cloudRegions: cloudRegions,
+ // hideModal:onCloseModal(),
+  //isModalOpen:isModalOpen
+};
+QuotaLoading.storyName = "Checking quota for instance";
+
+export const CreationInProgress = Template.bind({});
+CreationInProgress.args = {
+  isKasTrial: true,
+  cloudProviders: cloudProviders,
+  quota: { data: undefined, loading: false, isServiceDown: false },
+  kasQuota: { allowed: 60, consumed: 20, remaining: 40 },
+  kasTrial: { allowed: 60, consumed: 20, remaining: 40 },
+  hasKafkaCreationFailed: false,
+  userHasTrialInstance: false,
+  loadingQuota: false,
+  isCreationInProgress: true,
+  kafkaRequest: {
+    cloud_provider: {
+      value: "aws",
+    },
+    multi_az: {
+      value: true,
+    },
+    region: {
+      value: "us-east-1",
+    },
+    name: {
+      value: "suyash-test",
+    },
+  },
+  formSubmitted: false,
+  cloudRegions: cloudRegions,
+  //hideModal:onCloseModal(),
+  //isModalOpen:isModalOpen
+};
+CreationInProgress.storyName = "Creation in progress";
```

Comment:
Shouldn't the form be disabled while we are creating the instance?
Cancel and close button too.